dt/ospdo_dt.md

```diff
@@ -0,0 +1,43 @@
+    | vm    | 3      |
```

Comment:
Would you need compact Openshift source and destination?
Manila is configured with `ceph`, I believe you would need 3 additional nodes for Ceph?